haskell/test-common/Lorentz/Contracts/Test/Common.hs

```diff
@@ -211,6 +212,14 @@ constructSingleTransfer
 constructSingleTransfer from to amount
     = [(from, (#txs .! [(to, (#token_id .! 0, amount))]))]
 
+-- | Token metadata parameter that takes list of token ids and
+-- assumes that each of which resulting metadata is always valid
+assertAnyTokenMetaData :: [Natural] -> FA2.TokenMetaDataParam
+assertAnyTokenMetaData ids =
```

Comment:
I am not sure if it is proper to call this an assertion, because it can be something else, like posting token metadata to a different contract. right?